agent/ebs/watcher.go

```diff
@@ -131,6 +175,66 @@ func (w *EBSWatcher) overrideDeviceName(foundVolumes map[string]string) {
 	}
 }
 
+// assumes CSI Driver Managed Daemon is running else call will timeout
+func (w *EBSWatcher) StageAll(foundVolumes map[string]string) error {
+	for volID, deviceName := range foundVolumes {
+		// get volume details from attachment
+		ebsAttachment, _ := w.agentState.GetEBSByVolumeId(volID)
+		if ebsAttachment.IsSent() {
+			log.Warnf("State change event has already been emitted for EBS volume: %v.", ebsAttachment.EBSToString())
+			return nil
+		}
+		if ebsAttachment.HasExpired() {
+			log.Warnf("EBS status expired, no longer tracking EBS volume: %v.", ebsAttachment.EBSToString())
+			return nil
+		}
+		if ebsAttachment.IsAttached() {
+			return nil
```

Comment:
nit(non-blocking): perhaps we can also log that the attachment has already been attached on a debug level

Reply:
I removed this log because it gets very noisy.  adding this back and will update all to Debug level